homework-classes/App.js

```diff
@@ -43,19 +50,24 @@ class App {
   /**
    * Fetch contributor information for the selected repository and render the
    * repo and its contributors as HTML elements in the DOM.
-   * @param {object} repo The selected repository object
+   * @param {object} repository The selected repository object
    */
-  async selectRepository(repo) {
+  async selectRepository(repository) {
     try {
       this.clearContainer();
-      const contributors = await repo.fetchContributors();
+      const contributors = await repository.fetchContributors();
 
-      const repoContainer = Util.createAndAppend('div', this.mainContainer);
-      const contributorContainer = Util.createAndAppend('div', this.mainContainer);
+      const repoContainer = Util.createAndAppend('div', this.mainContainer, {
+        id: 'table-container',
+      });
+      const contributorContainer = Util.createAndAppend('div', this.mainContainer, {
+        class: 'Contributors-list',
```

Comment:
Use **kebab-case** for CSS names (lowercase letters and hyphens): `contributors-list`.